test/api_system_metrics_test.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/environment"
+
+	"github.com/go-check/check"
+)
+
+// APIEngineVersionMetricsSuite is the test suite for EngineVersion metrics API.
+type APIEngineVersionMetricsSuite struct{}
+
+func init() {
+	check.Suite(&APIEngineVersionMetricsSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *APIEngineVersionMetricsSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// SetUpSuite does common setup in the beginning of each suite.
+func (suite *APIEngineVersionMetricsSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// TearDownSuite run after each suite to do cleanup work for the whole suite.
+func (suite *APIEngineVersionMetricsSuite) TearDownSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// TestEngineVersionMetrics tests metrics of EngineVersion.
+func (suite *APIEngineVersionMetricsSuite) TestEngineVersionMetrics(c *check.C) {
+	key := "engine_daemon_engine_info"
+	hasMetric := HasMetric(c, key)
+	c.Assert(hasMetric, check.Equals, true)
```

Comment:
I'm afraid `hasMetric=true` isn't complete. How about to assert the value of label `kernel`?

Reply:
add matching label checks